setup.py

```diff
@@ -21,6 +21,19 @@
 site.ENABLE_USER_SITE = "--user" in sys.argv[1:]
 
 
+def cpp_path():
+    return os.environ.get("CC", "cpp")
```

Comment:
```suggestion
    return os.environ.get("CC", "cc")
```

Then you don't need special conditions. Though normally it's `cc -e` as a single argument, but I guess pycparser has a really bad design.

Reply:
Oh, and remember that `CC` can actually contain options, so unless pycparser can handle that, it would probably make sense to `shlex.split()` it first.

Reply:
> Then you don't need special conditions. Though normally it's `cc -e` as a single argument, but I guess pycparser has a really bad design.

@salahcoronya, how do you feel about this? I'm happy with the patch as is now, but do you want to change it?

Reply:
I just pushed the improved version moments before you commented.